### PR TITLE
.md linting pr

### PR DESCRIPTION
**Pull request by Mooshimi**
## About The Pull Request

Pacify the linter gods for some .md files

## Why It's Good For The Linters

Everyone deserves to be happy...
Also I think you guys should change your linter config to allow hard-tabbing (without a warning I mean) in these. Also two of the links in here are just not in existence not sure if you want me to just take them out all together or what

## Changelog

I don't think this is even noticeable unless you want me remove the bad links

### CODE REVIEW

**Comment by Mothblocks** [comment]
What linter? If we're going to accept a PR like this then whatever linter this is using should be in CI to prevent regressions

File: README.md [hunk 1]
```diff
@@ -7,25 +7,26 @@
...
 ## DOWNLOADING
+
 [Downloading](.github/guides/DOWNLOADING.md)
 
 [Running on the server](.github/guides/RUNNING_A_SERVER.md)
 
 [Maps and Away Missions](.github/guides/MAPS_AND_AWAY_MISSIONS.md)
 
-## :exclamation: How to compile :exclamation:
+## :exclamation: How to compile :exclamation
```
Comment (Mothblocks) [inline README.md:29]: wut?

File: tgui/README.md [hunk 2]
```diff
@@ -100,21 +101,21 @@ Remember to always run a full build of tgui before submitting a PR, because it c
 
 ## Troubleshooting
 
-**Development server is crashing**
+**Development server is crashing**:
```
Comment (Mothblocks) [inline tgui/README.md:104]: Shouldn't these all be `###` headers? This is going to look silly

File: .github/guides/VISUALS.md [hunk 3]
```diff
@@ -77,10 +77,10 @@ Somewhat like this
 // Lie to the compiler, and pretend we are using a mutable appearance
 // This lets us access vars as expected. Be careful with it tho
 /proc/mutate_icon_state(mutable_appearance/thing) 
-	var/mutable_appearance/temporary_lad = new()
-	temporary_lad.appearance = thing
-	temporary_lad.icon_state += "haha_owned"
-	return temporary_lad.appearance
+ var/mutable_appearance/temporary_lad = new()
```
Comment (Mothblocks) [inline .github/guides/VISUALS.md:80]: You mention whatever linter we're apparently using doesn't support hard tabs...we need to fix that before we can ever merge a PR that does something like this

**Comment by Mooshimi** [comment]
> What linter? If we're going to accept a PR like this then whatever linter this is using should be in CI to prevent regressions

https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md024

File: README.md (hunk 1 above)
Comment (Mooshimi) [reply README.md:29]: auto removal of trailing punctuation 1 moment

File: tgui/README.md [hunk 4]
```diff
@@ -127,16 +128,16 @@ To fix this kind of problem, run `bin/tgui --clean` and try again.
...
 development only features.
 
-**Debug Logs.**
+**Debug Logs.**:
 When running server via `bin/tgui --dev --debug`, server will print debug
 logs and time spent on rendering. Use this information to optimize your
 code, and try to keep re-renders below 16ms.
 
-**Kitchen Sink.**
+**Kitchen Sink.**:
```
[the remark this answers is not in the document]
Comment (Mooshimi) [reply tgui/README.md:136]: kk

**Comment by Mothblocks** [comment]
Do we actually use/recommend this internally or did you just run it over? It's fine if the latter answer is yes but if the former is also yes then this is just going to regress

File: tgui/README.md (hunk 2 above)
Comment (Mooshimi) [reply tgui/README.md:104]: Yeah I can change its structure if you want

File: .github/guides/VISUALS.md (hunk 3 above)
Comment (Mooshimi) [reply .github/guides/VISUALS.md:80]: it's a setting it says under the warning

**Comment by Mothblocks** [comment]
not stupid

**Comment by Mooshimi** [comment]
> not stupid

tldr i had this random extension without noticing Name: markdownlint
Id: DavidAnson.vscode-markdownlint
Description: Markdown linting and style checking for Visual Studio Code
Version: 0.48.1
Publisher: David Anson
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

**Comment by Mothblocks** [comment]
that is a reasonable messup but not stupid
